packages/pulumi-aws/src/apps/website/WebsiteApp.ts

```diff
@@ -136,6 +137,7 @@ export const WebsiteApp = defineApp({
             // Among other things, this determines the amount of information we reveal on runtime errors.
             // https://www.webiny.com/docs/how-to-guides/environment-variables/#debug-environment-variable
             DEBUG: String(process.env.DEBUG),
+            VARIANT: app.ctx.variant || "",
```

Comment:
Should we maybe be a bit more explicit here? Should we maybe use something like "STAGED_ROLLOUTS_VARIANT" or similar?

Reply:
👍